hail/python/hail/utils/java.py

```diff
@@ -40,7 +40,14 @@ def hc():
             backend_name = os.environ.get('HAIL_QUERY_BACKEND', 'spark')
             if backend_name == 'service':
                 from hail.context import init_service
-                init_service()
+                billing_project = os.environ['HAIL_BILLING_PROJECT']
+                if billing_project is None:
+                    raise ValueError('''\
+Cannot initialize Hail Query service backend without a billing
+project.  Call hl.init_service explicitly or set the
+HAIL_BILLING_PROJECT environment variable.
+''')
+                init_service(billing_project)
```

Comment:
I think you should just remove these changes, right? `ServiceBackend.__init__` will display a useful message.